main.go

```diff
@@ -323,17 +327,13 @@ func main() {
 		fmt.Println()
 		log.Infof("Updating ionic version to: %s", configs.IonicVersion)
 
-		if err := npmRemove(false, "ionic"); err != nil {
-			fail("Failed to remove ionic, error: %s", err)
-		}
-
-		if err := npmInstall(true, "ionic@"+configs.IonicVersion); err != nil {
+		if err := jspackage.Add(packageManager, true, "ionic@"+configs.IonicVersion); err != nil {
```

Comment:
did not you forget `npmRemove(false, "ionic")` ?

Reply:
Install global, remove local, do not install local